modules/webbrowser/src/interaction/cefinteractionhandler.cpp

```diff
@@ -101,8 +101,10 @@ void CEFInteractionHandler::handlePickingEvent(PickingEvent* p) {
             }
             bool mouseUp = MouseState::Release & mouseEvent->state() ? true : false;
             int clickCount = MouseState::DoubleClick & mouseEvent->state() ? 2 : 1;
```

Comment:
should move those two lines into the if statement

Reply:
Good, catch. See fix